Testscripts/Linux/TestRDMA_MultiVM.sh

```diff
@@ -270,6 +271,69 @@ function Run_IMB_NBC() {
 	fi
 }
 
+function Run_IMB_P2P() {
+	total_attempts=$(seq 1 1 $imb_p2p_tests_iterations)
+	imb_p2p_final_status=0
+	if [[ $imb_p2p_tests == "all" ]]; then
+		extra_params=""
+	else
+		extra_params=$imb_p2p_tests
+	fi
+	for attempt in $total_attempts; do
+		retries=0
+		while [ $retries -lt 3 ]; do
+			case "$mpi_type" in
+				ibm)
+					LogMsg "$mpi_run_path -hostlist $modified_slaves:$VM_Size -np $(($VM_Size * $total_virtual_machines)) -e MPI_IB_PKEY=$MPI_IB_PKEY $imb_p2p_path $extra_params"
+					$mpi_run_path -hostlist $modified_slaves:$VM_Size -np $(($VM_Size * $total_virtual_machines)) -e MPI_IB_PKEY=$MPI_IB_PKEY $imb_p2p_path $extra_params > IMB-P2P-AllNodes-output-Attempt-${attempt}.txt
+				;;
+				open)
+					LogMsg "$mpi_run_path --allow-run-as-root --host $master,$slaves -n $(($p2p_ppn * $total_virtual_machines)) $mpi_settings $imb_p2p_path $extra_params"
+					$mpi_run_path --allow-run-as-root --host $master,$slaves -n $(($p2p_ppn * $total_virtual_machines)) $mpi_settings $imb_p2p_path $extra_params > IMB-P2P-AllNodes-output-Attempt-${attempt}.txt
+				;;
+				intel)
+					LogMsg "$mpi_run_path -hosts $master,$modified_slaves -ppn $p2p_ppn -n $(($VM_Size * $total_virtual_machines)) $mpi_settings $imb_p2p_path $extra_params"
+					$mpi_run_path -hosts $master,$modified_slaves -ppn $p2p_ppn -n $(($VM_Size * $total_virtual_machines)) $mpi_settings $imb_p2p_path $extra_params > IMB-P2P-AllNodes-output-Attempt-${attempt}.txt
```

Comment:
$modified_slaves variable is only available in IBM test cases, not Intel. Should change to $slaves

Reply:
fixed in 67eadb55a6cd62eb271249df2decdecad6729662